assets/core/route-controller-informer-clusterrole.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
shouldn't these files be under its own subfolder like the other components?

Reply:
^ this is just cosmetic, we can return to this later if needed

Reply:
Let's do a follow up PR for assets arrangement.